pkg/collector/container.go

```diff
@@ -63,5 +63,6 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 		Image:        image,
 		VolumeMounts: volumeMounts,
 		Args:         args,
+		Env:          otelcol.Spec.Env,
```

Comment:
Could you add a test that exercises this?

Reply:
Yes I can look at it.